src/mod2c_core/CMakeLists.txt

```diff
@@ -39,11 +39,25 @@ set(NOCMODL_CORE_SOURCES
 "simultan.c" "units.c"
 )
 
-bison_target(diffeq diffeq.y ${CMAKE_CURRENT_BINARY_DIR}/diffeq.c)
-bison_target(parse1 parse1.y ${CMAKE_CURRENT_BINARY_DIR}/parse1.c)
-
-flex_target(lex lex.l ${CMAKE_CURRENT_BINARY_DIR}/lex.c)
```

Comment:
I think @alkino won't be happy to see removal of `flex_target & bison_target` and re-appearance of verbose custom commands  :)